### PR TITLE
Add point type to replace task's points field

### LINKED ISSUE #33: Task points float value
Clickup provides ability to add custom value points and they can be float, so if you try to get task with float points it will return json unmarshal error
`json: cannot unmarshal number 0.5 into Go struct field Task.points of type int`

### PR DESCRIPTION
**Pull request by a-thug**
This PR resolves #33.
I have figured out that I can put float in Points field. So I supported so that We can get both Int and Float.

### CODE REVIEW

File: clickup/point.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	}
+	var f float64
+	if e := json.Unmarshal(b, &f); e == nil {
+		p.FloatVal = &f
+		return
+	}
+
+	p.Value = json.RawMessage(b) //default to raw message
+	return
+}
```
Comment (w-haibara) [inline clickup/point.go:79]: When a value cannot be unmarshaled to either int64 or float64, it should return an error as an illegal point value.
Then, assuming that either IntVal or FloatVal is not nil, other methods of type Point can be implemented. For example there.
```
if p.IntVal != nil {
    // process as a int value
} else {
    // process as a float value
}
```

**Review by w-haibara — COMMENTED** [review]
Thank you!
I made some comments, please check them out.

File: clickup/point.go (hunk 1 above)
Comment (w-haibara) [reply clickup/point.go:79]: `Int64()` / `Float64()` of `json.Number` can be used to convert Point.Value to int64/float64.

File: clickup/point.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	}
+
+	if p.FloatVal != nil {
+		val = strconv.FormatFloat(*p.FloatVal, 'f', -1, 64) //-1 will remove unimportant 0s
+		return
+	}
+
+	val = string(p.Value)
+	return
+}
```
[the remark this answers is not in the document]
Comment (a-thug) [reply clickup/point.go:28]: That's right!
Good!

**Review by w-haibara — APPROVED** [review]

**Pull request merged**